tube/etl/indexers/aggregation/parser.py

```diff
@@ -57,6 +57,8 @@ def __init__(self, mapping, model, dictionary):
         self.nest_leaves = []
 
         self.aggregated_nodes = []
+        if "filter" in self.mapping:
+            self.filter = create_filter_from_json(self.mapping.get("filter"))
```

Comment:
Maybe we should change this to 
```python
     self.filter = create_filter_from_json(self.mapping.get("filter")) if "filter" in self.mapping else None
```

Otherwise the etl process would fail at [this line](https://github.com/uc-cdis/tube/blob/33fc9b744818fbf100fe07e1cd3df337eb50ca5e/tube/etl/indexers/aggregation/translator.py#L546-L546) when there is no `filter` key in the etlMapping